toolkit/app-gen-toc.py

```diff
@@ -630,7 +630,7 @@ def updateDeviceConfig(file):
                 unwanted_item = True
         if unwanted_item:
             print(
-                "[ERROR] File '" + file + "' should not contain 'miscellaneous' entry"
+                "[ERROR] File '" + str(file) + "' should not contain 'miscellaneous' entry"
```

Comment:
The pattern I used elsewhere for this was `.as_posix()` at the assignment of the variable, so that would be:
```py
file = (paths.CONFIG_INPUT_DIR / file).as_posix()
```
on like 619 above.